tests/test_session.py

```diff
@@ -610,13 +610,8 @@ def test_general_exception_behaviour_in_session(new_solver_session):
         return_without_path=False,
     )
 
-    # Error in server:
-    # This appears to be a surface mesh.\nSurface meshes cannot be read under the /file/read-case functionality.
-    # with pytest.raises(RuntimeError, match="Surface meshes cannot be read") as exec_info:
-    #     solver.settings.file.read(file_type='case', file_name=mesh_file_2d)
-    # # Assert that exception is propagated from the Fluent server
-    # assert isinstance(exec_info.value.__context__, grpc.RpcError)
```

Comment:
Please include this assert and the match with the error message in line 615. That will confirm that the error is orignated from Fluent.

Reply:
Also can add a similar assert for the previous post-processing error

Reply:
Done. Thank you.